src/Microsoft.Health.Fhir.Shared.Api/Controllers/ConvertDataController.cs

```diff
@@ -76,6 +78,7 @@ public async Task<IActionResult> ConvertData([FromBody] Parameters inputParams)
             string inputData = ReadStringParameter(inputParams, ConvertDataProperties.InputData);
             string templateCollectionReference = ReadStringParameter(inputParams, ConvertDataProperties.TemplateCollectionReference);
             string rootTemplate = ReadStringParameter(inputParams, ConvertDataProperties.RootTemplate);
+            bool treatDatesAsStrings = ReadBoolParameter(inputParams, ConvertDataProperties.JsonDeserializationTreatDatesAsStrings);
```

Comment:
Just to confirm, this won't error if the request doesn't have this param present?